src/calculateRentalCost.js

```diff
@@ -29,6 +29,16 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const costs = 40;
+  const discountLow = 20;
+  const discountHigh = 50;
```

Comment:
inversing name parts looks inappropriate
```suggestion
  const lowDiscount = 20;
  const highDiscount = 50;
```